man/meson.build

```diff
@@ -1,42 +1,3 @@
-# SPDX-FileCopyrightText: Samsung Electronics Co., Ltd
```

Comment:
Is removing the Copyright intentional?

Reply:
Also is removing the manpages for all of these tools intentional, looks a little off...

Reply:
> Also is removing the manpages for all of these tools intentional, looks a little off...

They are not removed? They are updated and then there are additions for e.g. ``kvs``.